### DISCUSSION TITLE
Возможность собирать стату с нескольких репозиториев

### DISCUSSION THREAD

**Discussion by fdman-liis** [bakhirev/assayo#31] [Ideas]
Люди ротируются между проектами и интересно смотреть вовлеченность в целом при работе над несколькими репозиториями

**Comment by fdman-liis** [comment]
https://github.com/bakhirev/assayo-crawler

**Comment by bakhirev** [comment]
Да, все верно. Кравлер как раз и может по настройкам собирать разные подборки и комбинации по репозиториям. К нему так же есть (недописанный пока) UI, чтобы можно было все это накликать мышкой, а не читать конфиги. Демо: https://assayo.online/showcase/

К сожалению в данный момент на нем минимальный фокус, т.к. нет интереса от целевой аудитории и заказа. Если будет интерес или заказ - то он будет активнее развиваться

**Comment by bakhirev** [comment]
@fdman-liis 
Создал Docker образ для этой задачи https://hub.docker.com/r/bakhirev/assayo-crawler
Обновил код и документацию https://github.com/bakhirev/assayo-crawler/
Приглашаю на тестирование, чтобы понять, как сделать его более понятным / доступным.
